src/snowflake/snowpark/session.py

```diff
@@ -722,12 +722,24 @@ def _resolve_packages(
             package_name = package_req.key
             if validate_package:
                 if package_name not in valid_packages:
+                    is_anaconda_terms_acknowledged = self._run_query(
+                        "select system$are_anaconda_terms_acknowledged()"
+                    )[0][0]
+                    if is_anaconda_terms_acknowledged:
+                        detailed_err_msg = (
+                            "it is not available in Snowflake. Check information_schema.packages "
+                            "to see available packages for UDFs. If this package is a "
+                            '"pure-Python" package, you can find the directory of this package '
+                            "and add it via session.add_import()."
```

Comment:
How about adding this link in the message? 
https://docs.snowflake.com/en/developer-guide/snowpark/python/creating-udfs.html#using-third-party-packages-from-anaconda-in-a-udf

Reply:
added